packages/apps/spaces/app/organization/[id]/src/components/OrganizationSidenav/OrganizationSidenav.tsx

```diff
@@ -57,29 +59,41 @@ export const OrganizationSidenav = () => {
       borderRadius='2xl'
       borderColor='gray.200'
     >
-      <Tooltip label={data?.organization?.name} placement='bottom'>
-        <Flex gap='2' align='center' mb='4'>
-          <IconButton
-            size='xs'
-            variant='ghost'
-            aria-label='Go back'
-            onClick={() => {
-              router.push(`/${lastActivePosition?.root || 'organization'}`);
-            }}
-            icon={<Icons.ArrowNarrowLeft color='gray.700' boxSize='6' />}
-          />
+      <Flex gap='2' align='center' mb='4'>
```

Comment:
Aria-label is for screen readers, it won't result in displaying tooltip which in this case is needed when text is truncated